# Add workaround for wrong code generation on m68k (leading to
# miscompilation of bash and coreutils at -O1 or -O2).

--- gcc/loop.c	21 Dec 2002 20:25:40 -0000	1.389.2.7.2.7
+++ gcc/loop.c	27 Mar 2003 09:50:18 -0000
@@ -1154,7 +1154,7 @@
 
   /* Now that we've moved some things out of the loop, we might be able to
      hoist even more memory references.  */
-  load_mems (loop);
+  /*load_mems (loop);*/
 
   /* Recalculate regs->array if load_mems has created new registers.  */
   if (max_reg_num () > regs->num)
